course_catalog/factories.py

```diff
@@ -341,3 +342,23 @@ def topics(self, create, extracted, **kwargs):
 
     class Meta:
         model = UserList
+
+
+class VideoResourceFactory(LearningResourceFactory):
+    """Factory for VideoResource"""
+
+    video_id = factory.Sequence(lambda n: "VIDEO-%03d.MIT" % n)
+    platform = FuzzyChoice([PlatformType.youtube.value])
```

Comment:
Is this a `FuzzyChoice` because other video platforms will be added eventually?

Reply:
Yeah, just figured this will make that change slightly easier